src/components/new-request/search.vue

```diff
@@ -107,7 +107,7 @@
           :type="'request a name for'"
           :showDialog="showSocietiesInfo"
         />
-        <template v-if="isNameChangeable && !isSocietyDisabled">
+        <template v-else-if="isNameChangeable">
```

Comment:
This whole section only needs to be displayed if SocietiesInfo wasn't shown, hence the else. And I left the other condition in there.